docs/usage/README.md

```diff
@@ -0,0 +1,125 @@
+
+
+Kubernetes operators can benefit from different capabilities like accessing the logs for 
+already terminated containers, perform fast and sophisticated search for investigating long 
+lasting problems or recurring problems based on logs from a long period of time. 
+
+Below in this guide, you will find the procedure how to access the log data for your clsuters.
+
+## Procedure
+
+Use this procedure to view the log data of your Kubernetes cluster.
+
+### Accessing Grafana
+1. Open **Gardener dashboard** and navigate to the cluster which log data you want to view
+2. Hover your mouse over cluster name to open and view the details for the cluster. Find the **Logging and Monitoring** section and use the link and credentials to log-in to Grafana.
+
+![Navigate to Logging and Monitoring Tile](images/gardener-dashboard-logging.png)
+
+### Using Grafana
+
+There are two ways for tracking log messages in Grafana.
+
+#### Predefined Dashboards
+The first option is to use the predefined dashboards.
+Go to the **Home** menu and after that you can choose which dashboard to open. The dashboards which contain log visualizations for the different Grafana deployments are:
+
+  * Garden Grafana
+    * Pod Logs
+    * Extensions
+    * Systemd Logs
+  * User Grafana
+    * Kubernetes Control Plane Status
+  * Operator Grafana 
+    * Kubernetes Pods
+    * Kubernetes Control Plane Status
+
+![](images/dashboards.png)
+
+#### Explore menu
+The second option is to use the **Explore** menu.
+The first thing for enabling this option is to authenticate in front of the Grafana UI.
+This happens with the button in the bottom left corner
+![](images/login-button.png)
+
+The default credentials for the Grafana are:
+`username` : `admin`
+`password` : `admin`
+![](images/login-credentials.png)
+
+On successful login you will be asked for changing the default password. Please ***do not*** do that because these credentials will be used from all of the operators so this step ***should be skipped***.
+
+![](images/skip-password-change.png)
+
+After that just select the ***Explore*** (Left side of the screen) menu.
+
+![](images/explore-logs.png)
+
+There you can create a custom log filters based on the predefined labels used in `Loki`
+
+The following properties can be managed in the `Explore` menu:
+
+1) `Datasource` (top left corner) should be set on Loki
+2) `Timerange` (top right corner) is used to filter logs over a different period of time
+3) `Label Selector` (top left corner) is used to filter logs based on the `Loki`'s labels and their values. 
+For example: 
+`pod_name="kube-apiserver-1234-1234"` or you can use a regex: `pod_name=~"kube-apiserver.+"`
+4) `Severity` (left side of the screen). This option is used to filter log messages with specific severity.
+
+Lastly click on **Run Query** (top right corner) and the log messages which fit on the properties listed above will be displayed.
+
+## Using log data for a problem resolution
```

Comment:
Having a realistic hands on example is plausible. But the idea here is to show how to use the logging to solve this problem, i.e. how to find out the hint among the log messages. And precisely that part is missing.

My proposal is to move this sample problem up, and then use it as a case study to describe the two methods to explore logs described above. Or do that here, but this will make the material quite longer I think.

Reply:
This is too short and straightforward documentation. If I move up the Example should this be enough?

Reply:
No, the point is to describe what you do with the logging tools to solve an issue.